src/config/global_vars.py

```diff
@@ -246,16 +246,12 @@ def init():
     global virtual_serial_port
     global robot_serial_port
 
-    spi = board.SPI()   # MOSI pin 19
     global pixels
-    # Board Setup
-    pixels = neopixel.NeoPixel_SPI(
-        spi,                            # SPI object
-        24,                             # Number of LEDs (Num of Pixels)
-        brightness = 0.05,              # Scale from 0.00 to 1.00 (Higher = Brighter), CAUTION: 1.00 hurts your eyes
-        pixel_order = neopixel.GRB      # G and R are reversed, so the colors are actually in order of RGB
-    )
-    
-    
-    
-
+    pixels = None
+    if robot == "pi":
```

Comment:
lets choose one spot to setup neopixel for spi or regular. do it in the same spot, not two diff files. id prefer to do it in globals tbh